src/main/java/hudson/plugins/active_directory/ActiveDirectoryUnixAuthenticationProvider.java

```diff
@@ -330,7 +330,7 @@ public UserDetails call() throws AuthenticationException {
                         closeQuietly(context);
                     }
                 }
-            });
+            }).updateUserInfo();
```

Comment:
this will now cause the user info to be updated always. Previously we only updated the info if the value was not in the cache